### PR TITLE
Add lead breadcrumb variation

### PR DESCRIPTION
**Pull request by LukasHirt**
## Motivation:
Enable usage of breadcrumbs as a title of page.

## Screenshot:
image

### CODE REVIEW

File: src/styles/theme/oc-breadcrumb.scss [hunk 1]
```diff
@@ -35,4 +35,11 @@
       }
     }
   }
+
+  &-lead &-list-item {
+    &::before, a, span {
```
Comment (micbar) [inline src/styles/theme/oc-breadcrumb.scss:40]: Codacy found an issue: [Expected newline after "," (selector-list-comma-newline-after)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5792675)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5792675&bid=18961514)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5792675&bid=18961514)

File: src/styles/theme/oc-breadcrumb.scss [hunk 2]
```diff
@@ -35,4 +35,13 @@
       }
     }
   }
+
+  &-lead &-list-item {
+    &:before,
```
Comment (micbar) [inline src/styles/theme/oc-breadcrumb.scss:40]: Codacy found an issue: [Expected double colon pseudo-element notation (selector-pseudo-element-colon-notation)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5792675)

**Comment by LukasHirt** [comment]
@kulmann Added the semicolon to make codacy happy. But the review got again dismissed automatically *sigh* Can you pls re-review? 🙏

**Pull request merged**